projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -429,6 +429,7 @@ if (environment.cpq) {
         a11yWrapReviewOrderInSection: true,
         enableCarouselCategoryProducts: true,
         enableSecurePasswordValidation: true,
+        enableClaimCustomerCouponWithCodeInRequestBody: true,
```

Comment:
That is true that we want to enable all toggles in our demo app, but in this exact case, if I understand correctly, the API will be available starting from some future version. 
If yes - we toggle set to true, our e2e tests will be broken because this functionality will not work before the servers update.
In this case i think it should be false.
I also thinking, maybe it is good to add this information to the toggle JSdoc? Some info telling that if you enabling this toggle you should have cx in version FP8

Reply:
Thanks,  We integrated the new endpoints into Commerce since 2211.28, see following comment.

Reply:
Ok, in this case it could be true, agree.
I run all e2e tests against your branch, and there are couple of e2e tests in different libs that start failing after this change.
Here you can check the result https://cloud.cypress.io/projects/k3nmep/runs/46437/overview?runStatus=FAILED&runStatus=CANCELLED&runStatus=ERRORED&runStatus=PASSED&runStatus=RUNNING&runStatus=NOTESTS&runStatus=TIMEDOUT&roarHideRunsWithDiffGroupsAndTags=1
![image](https://github.com/user-attachments/assets/1ad7dd40-8725-4059-990e-791b54eaf112)


The first one "Order history" is flaky and probably not related to your changes, it failed already earlier, but the other ones seems to be related to coupon.

Can you check it please on your side?

Reply:
thanks, sure, will check and update

Reply:
Hi @rmch91 , I have checked the failed cypress test  cases except "Order history" which think not related my changes, yeah, the other failed cases is related with my changes,  so I have updated the case with a little changes, now after update local running shows it is ok:
![image](https://github.com/user-attachments/assets/eee217f3-42ce-45a7-b161-d643517ddc47)

I have no access to trigger the whole e2e cypress tests, could you help re-run the whole e2e cypress tests based on the fix branch after the cases updated? 
Thanks a lot.

Reply:
Many thanks, I run it and waiting for the result

Reply:
Two of them still failing

https://cloud.cypress.io/projects/k3nmep/runs/46446/overview?runStatus=FAILED&runStatus=CANCELLED&runStatus=ERRORED&runStatus=PASSED&runStatus=RUNNING&runStatus=NOTESTS&runStatus=TIMEDOUT&roarHideRunsWithDiffGroupsAndTags=1

Reply:
Hi @rmch91 , thanks for information,
The cypress cases still failed because the CI server has not deployed the commerce cloud with the new occ endpoint(though 2211.28 already has the endpoint),and CI server  deployed old versions of commerce, and it reports '404' error when claim coupon with new method, like followings shows(It explains why the test cases local ok, but failed in CI server(20.83.184.244:9002)).
![image](https://github.com/user-attachments/assets/360829c5-80c4-4914-a074-d0435aff428e)

And I do not know when deploy new version commerce, to avoid break CI e2e cypress tests(thanks for early remind), I disable the new added toggle to be 'false', add comments about versions in JSdoc and also skip my new added e2e cypress tests with comments.  Now the whole e2e cypress tests passed after rerun the pipeline.
Also I have questions here:
when do we enable the skip test cases? do we need to create jira tickets to follow it?
Also please check and review again, thanks a lot.